src/entities/euring-codes/age-entity.ts

```diff
@@ -18,11 +19,17 @@ export class Age implements Dictionary {
   @Column('varchar', { nullable: true, default: null })
   public desc_byn: string | null;
 
-  @OneToMany(() => Ring, m => m.ageScheme)
-  public ring: Ring[];
+  @OneToMany(() => Ring, m => m.ageMentioned)
+  public ringMentioned: Ring[];
```

Comment:
this naming and the similar below are confusing, would you change it to something like
`mentionedInRing`, `concludedInRing` and so on?

Reply:
Yes, I agree with you, it will be more clear. Done.